2022/02/index.js

```diff
@@ -2,13 +2,93 @@ const { promises: fs } = require('fs');
 
 async function getInput() {
   const input = (await fs.readFile(`${__dirname}/input.txt`))
```

Comment:
This looks overly copy/pasty, any chance for DRYness between questions?

Reply:
I didn't want to bother =P 🤔